src/login/login.cpp

```diff
@@ -718,8 +719,12 @@ void log_init(int argc, char** argv)
         {
             logFile = argv[i + 1];
         }
+	if (strcmp(argv[i], "--append-date") == 0)
+	{
+	    appendDate = true;
+	}
```

Comment:
Based on your feedback, would it not make more sense to roll this flag into the `--log` arg, so that you get the name you specified without having to use an additional flag?

Reply:
I don't think I follow you. Can you show me an example of what a complete command would look like?

`./topaz_game --ip 159.250.1.60 --port 54230 --log /home/ganiman/game-54230.log --append-date`

Please clarify what you mean for me and I'll see if it makes sense to me.

Reply:
@zach2good I think its fine and even a little more intuitive to me the way he has it. Especially in linux land this is how this type of thing generally looks and even when the build only worked on windows way back when, for some reason the original devs chose linux style arguments. Having one arg with 2 inputs following it can actually be more confusing to people.